api/v1/db/oracledb/game-dao.js

```diff
@@ -145,6 +146,41 @@ const validateGame = async (id) => {
   }
 };
 
+const cleanTableCardsByGameId = async (gameId) => {
+  const connection = await conn.getConnection();
+  try {
+    const sqlParams = [gameId];
+    const cleanCardSqlQuery = `
+    DELETE FROM TABLE_CARDS WHERE GAME_ID = :gameID
+    `;
+    await connection.execute(cleanCardSqlQuery, sqlParams, { autoCommit: true });
+  } finally {
+    connection.close();
+  }
+};
+
+
+const deleteGameByGameId = async (gameId) => {
+  const connection = await conn.getConnection();
+  try {
+    await cleanTableCardsByGameId(gameId);
+    await playerDao.deletePlayersByGameId(gameId);
```

Comment:
Similar to my other comment, these other sql queries will need to be ran without being committed to make sure the request is atomic